.circleci/config.yml

```
@@ -216,6 +220,40 @@ jobs:
               SCALINGO_APP_TOKEN=sm://${GCP_PROJECT}/${APP_NAME}_scalingo_app_token" \
             --image=${GCP_REGION}-docker.pkg.dev/${GCP_PROJECT}/${GCP_REGISTRY_NAME}/${IMAGE_NAME}:latest
 
+  deploy-cron-and-worker:
+    executor: helm
+    environment:
+      GCP_REGION: europe-west1
+      GKE_CLUSTER_NAME: cluster-metier-development
+      HELM_EXPERIMENTAL_OCI: 1
+      CHART_REGISTRY_SERVER: europe-west1-docker.pkg.dev/pass-culture-app-projet-test
+      CHART_REGISTRY_NAMESPACE: pass-culture-artifact-registry
+      CHART_REGISTRY_NAME: passculture-chart
+      HELM_CHART_URI: ${CHART_REGISTRY_SERVER}/${CHART_REGISTRY_NAMESPACE}/${CHART_REGISTRY_NAME}:${TAG}
```

On pourrait reprendre le même exemple que pour les autres jobs ?
```suggestion
      HELM_CHART_URI: ${GCP_REGION}-docker.pkg.dev/${GCP_REGISTRY_NAME}/${CHART_REGISTRY_NAMESPACE}/${CHART_REGISTRY_NAME}:${TAG}

```